config.yaml

```diff
@@ -84,3 +84,8 @@ options:
     default:
     description: |
       The secret key for the Auth0 app client to which tokens will be issued.
+  deploy-env:
```

Comment:
Can you add it to the `license-manager-agent.yaml` example in the README, please?

Reply:
yes, I will